do not poke around /usr/include and /usr/lib directly as it's (1) not needed
and (2) breaks cross-compiling

patch from Chromium OS

--- a/setup.py
+++ b/setup.py
@@ -147,7 +147,6 @@ class pil_build_ext(build_ext):
             add_directory(library_dirs, "/opt/local/lib")
             add_directory(include_dirs, "/opt/local/include")
 
-        add_directory(library_dirs, "/usr/local/lib")
         # FIXME: check /opt/stuff directories here?
 
         prefix = sysconfig.get_config_var("prefix")
@@ -207,13 +206,6 @@ class pil_build_ext(build_ext):
             if os.path.isfile(os.path.join(tcl_dir, "tk.h")):
                 add_directory(include_dirs, tcl_dir)
 
-        # standard locations
-        add_directory(library_dirs, "/usr/local/lib")
-        add_directory(include_dirs, "/usr/local/include")
-
-        add_directory(library_dirs, "/usr/lib")
-        add_directory(include_dirs, "/usr/include")
-
         #
         # insert new dirs *before* default libs, to avoid conflicts
         # between Python PYD stub libs and real libraries
